src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/barcode/BarcodeService.kt

```diff
@@ -48,4 +53,8 @@ class BarcodeService(
         }
       }
   }
+
+  private fun BarcodeRecipient.lookupRecipient() {
```

Comment:
Now I see this in the light of day, an extension function just for this is probably overkill (I originally thought it might be doing a bit more, but at the moment its not). I'll improve this in my next PR